src/main/resources/templates/cssFiles/headerCss.css

```diff
@@ -0,0 +1,54 @@
+body{
```

Comment:
I think you should name the folder simply `css` or `styles`. It's not professional to state the obvious. It's clear that the folder will contain files. 🙂

Reply:
Also, it's more professional to name this file `header.css`.

Reply:
Renamed the folder in order to be more professional :D
Renamed the CSS file from headerCss.css into header.css